scripts/build-tests/link-theatre-packages.mjs

```diff
@@ -0,0 +1,40 @@
+/**
+ * Add the theatre packages to the build test projects with `yalc link`
+ */
+
+import path from 'path'
+import {colorize, getTestBuildProjects} from './utils.mjs'
+
+const root = path.normalize(path.join(__dirname, '..', '..'))
+const buildTestProjectAbsPaths = getTestBuildProjects(root)
+
+// The packages that should be linked with `yalc`
+const packagesToLink = ['@theatre/core', '@theatre/studio']
```

Comment:
Any reason we're not also linking `@theatre/dataverse`?

Reply:
Also, `@theatre/r3f`?

Reply:
They weren't necessary for the  build tests before. 

(But now I had to add them for the `create-react-app-r3f` project (053e3d09), so I'll resolve this conversation! :relaxed:)